WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -64,5 +75,22 @@ public void Initialize()
 				Title += $" - {Network}";
 			}
 		}
+
+		// void IDialogHost.ShowDialog<TDialog>(TDialog dialogViewModel)
+		// {
+		// 	var dialogHost = (this as IDialogHost);
+		// 	dialogHost.CurrentDialog = dialogViewModel;
+		// 	DialogStateListener(true);
+		// }
+
+		// void IDialogHost.CloseDialog()
+		// {
+		// 	DialogStateListener(false);
+		// }
+
+		// public void SetDialogStateListener(Action<bool> listener)
+		// {
+		// 	this.DialogStateListener = listener;
+		// }
```

Comment:
Should this be removed?

Reply:
@yahiheb im going to ask @jmacato to mark this as WIP, its no where ready for code standard review just yet.